linode/manager
# M3-209 - Component UI Tests - Part Deux

* Created tests for HelpIcon, ErrorState, IconTextLink, Pagination Controls, Linear Progress, Circle Progress tests
* Added `e2e:all` script for running manager tests and storybook tests on docker-compose.
* Added waitForFocus utility method to set focus to a frame and check if an element exists continuously until element exists. 

### To Test with Docker:
`yarn docker:e2e` (Make sure you've set your `.env` file to use the appropriate `REACT_APP_CLIENT_ID='DOCKER-CLIENT-ID`) must redirect to `https://manager-local:3000/oauth/callback`

This will test the `e2e:all` yarn script.

### To Test Locally:
`yarn start` (must be running your local dev environment)

New shell:
`yarn selenium`

New Shell:
`yarn e2e`
